src/components/Input/Input.styled.ts

```diff
@@ -11,16 +12,19 @@ const INPUT_WIDTH: Record<InputProps['size'], string> = {
 const Input = styled.input<InputProps>`
   display: block;
   width: ${({ size }) => INPUT_WIDTH[size]};
-  border: 1px solid #000000;
+  border: 1px solid ${theme.color.sub1};
```

Comment:
```suggestion
  border: ${({theme}) => 1px solid ${theme.color.sub1}};
```
아마 이런식으로 되는 걸로 알고있어요